### PR TITLE
fix em description

### LINKED ISSUE #35297: em is relative to element's own font size, not parent
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units

### What specific section or headline is this issue about?

_No response_

### What information was incorrect, unhelpful, or incomplete?

The article mentions in multiple times that "em is relative to the font size of the parent element". This is in accurate for properties other than font-size.

### What did you expect to see?

em uses the element's own font size

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/css/building_blocks/values_and_units`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/css/building_blocks/values_and_units/index.md
* Last commit: https://github.com/mdn/content/commit/44b18841ff739fbf1a5450805d85f839fa3e68a5
* Document last modified: 2024-07-25T20:16:09.000Z

</details>

### PR DESCRIPTION
**Pull request by gagan-bhullar-tech**
### Description

PR fixes #35297

### CODE REVIEW

**Review by Josh-Cena — CHANGES_REQUESTED** [review]
This is not what the issue asked for. You are changing the definition of `rem`, which is correctly "the root element". However there are multiple places on this page that referred to "the parent element" when talking about `em` (including here) and you need to fix those.

**Comment by gagan-bhullar-tech** [comment]
> This is not what the issue asked for. You are changing the definition of `rem`, which is correctly "the root element". However there are multiple places on this page that referred to "the parent element" when talking about `em` (including here) and you need to fix those.

@Josh-Cena thanks. Updated PR.

File: files/en-us/learn/css/building_blocks/values_and_units/index.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ Most of these units are more useful when used for print, rather than screen outp
 
 Relative length units are relative to something else. For example:
 
-- `em` and `rem` are relative to the font size of the parent element and the root element, respectively.
+- `em` and `rem` are relative to the font size of its own element and the root element, respectively.
```
Comment (Josh-Cena) [inline files/en-us/learn/css/building_blocks/values_and_units/index.md:156]:
```suggestion
- `em` is relative to the font size of this element, or the font size of the parent element when used for {{cssxref("font-size")}}. `rem` is relative to the font size of the root element.
```

File: files/en-us/learn/css/building_blocks/values_and_units/index.md [hunk 2]
```diff
@@ -166,7 +166,7 @@ In the example below, you can see how some relative and absolute length units be
...
 
-The third box uses `em` units. These are relative to the font size. I've set a font size of `1em` on the containing {{htmlelement("div")}}, which has a class of `.wrapper`. Change this value to `1.5em` and you will see that the font size of all the elements increases, but only the last item will get wider, as its width is relative to that font size.
+The third box uses `em` units. These are relative to its own element's font size. I've set a font size of `1em` on the containing {{htmlelement("div")}}, which has a class of `.wrapper`. Change this value to `1.5em` and you will see that the font size of all the elements increases, but only the last item will get wider, as its width is relative to that font size.
```
Comment (Josh-Cena) [inline files/en-us/learn/css/building_blocks/values_and_units/index.md:169]:
```suggestion
The third box uses `em` units. These are relative to the element's font size. I've set a font size of `1em` on the containing {{htmlelement("div")}}, which has a class of `.wrapper`. Change this value to `1.5em` and you will see that the font size of all the elements increases, but only the last item will get wider, as its width is relative to that font size.
```

File: files/en-us/learn/css/building_blocks/values_and_units/index.md [hunk 3]
```diff
@@ -180,7 +180,7 @@ The HTML illustrated below is a set of nested lists — we have two lists in tot
 
 To start with, we set 16px as the font size on the `<html>` element.
 
-**To recap, the em unit means "my parent element's font-size"** in the case of typography. The {{htmlelement("li")}} elements inside the {{htmlelement("ul")}} with a `class` of `ems` take their sizing from their parent. So each successive level of nesting gets progressively larger, as each has its font size set to `1.3em` — 1.3 times its parent's font size.
+**To recap, the em unit means "my own element's font-size"** in the case of typography. The {{htmlelement("li")}} elements inside the {{htmlelement("ul")}} with a `class` of `ems` take their sizing from their own element. So each successive level of nesting gets progressively larger, as each has its font size set to `1.3em` — 1.3 times its own element's font size.
```
Comment (Josh-Cena) [inline files/en-us/learn/css/building_blocks/values_and_units/index.md:183]:
```suggestion
**To recap, the `em` unit means "my parent element's font-size"** if used for `font-size` (and "my own font-size" when used for anything else). The {{htmlelement("li")}} elements inside the {{htmlelement("ul")}} with a `class` of `ems` take their sizing from their parent. So each successive level of nesting gets progressively larger, as each has its font size set to `1.3em` — 1.3 times its parent element's font size.
```

**Review by Josh-Cena — CHANGES_REQUESTED** [review]

File: files/en-us/learn/css/building_blocks/values_and_units/index.md [hunk 4]
```diff
@@ -180,7 +180,7 @@ The HTML illustrated below is a set of nested lists — we have two lists in tot
 
 To start with, we set 16px as the font size on the `<html>` element.
 
-**To recap, the em unit means "my parent element's font-size"** in the case of typography. The {{htmlelement("li")}} elements inside the {{htmlelement("ul")}} with a `class` of `ems` take their sizing from their parent. So each successive level of nesting gets progressively larger, as each has its font size set to `1.3em` — 1.3 times its parent's font size.
+**To recap, the `em` unit means "my parent element's font-size"** if used for `font-size` (and "my own font-size" when used for anything else). The {{htmlelement("li")}} elements inside the {{htmlelement("ul")}} with a `class` of `ems` take their sizing from their parent. So each successive level of nesting gets progressively larger, as each has its font size set to `1.3em` — 1.3 times its parent element's font size.
 
 **To recap, the rem unit means "The root element's font-size"** (rem stands for "root em"). The {{htmlelement("li")}} elements inside the {{htmlelement("ul")}} with a `class` of `rems` take their sizing from the root element (`<html>`). This means that each successive level of nesting does not keep getting larger.
```
Comment (Josh-Cena) [inline files/en-us/learn/css/building_blocks/values_and_units/index.md:185]:
```suggestion
**To recap, the `rem` unit means "The root element's font-size"** (rem stands for "root em"). The {{htmlelement("li")}} elements inside the {{htmlelement("ul")}} with a `class` of `rems` take their sizing from the root element (`<html>`). This means that each successive level of nesting does not keep getting larger.
```

**Review by Josh-Cena — APPROVED** [review]
Thanks, LGTM
